addons/kafka.md

```diff
@@ -150,6 +150,104 @@ Kafka requires TLS client certificates for authentication. Make sure your
 Kafka and/or TLS library can do client certificate authentication, and that you can
 specify a custom CA certificate for server validation.
 
+## Migrating from on-prem to Aiven
+
+There are multiple viable strategies for migrating to Aiven. Which one is best for your team depends
+on many factors. Here we will describe a few approaches, other approaches are also possible. You 
+need to find the one that suits your situation best. Make sure you understand how the different 
+approaches affects your message processing. Some approaches might result in messages being processed 
+more than once, and some approaches might allow for messages to be processed out of order.
+
+If you are using the Schema Registry, you need to make sure that your schemas are uploaded to the 
+new schema registry in Aiven when migrating. How you do that depends on how you do your migration.
+
+The first step is always:
+
+ 0. Create the topic using the new Topic resource, and set proper ACLs
```

Comment:
maybe a link to an example? (howto Topic resource?)

Reply:
Fixed! 👍